packages/react-ai/src/types.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  Conversation as SDKConversation,
```

Comment:
Do we want to take a dependency on these types?

Reply:
I think it might be easier then copying them all over, but i wanted to do it this way so it'd be easy to swap